test/helpers/cons.go

```diff
@@ -28,6 +29,8 @@ import (
 var (
 	// HelperTimeout is a predefined timeout value for commands, in seconds.
 	HelperTimeout int64 = 300
+	// HelperTimeoutDuration is a predefined timeout value for commands.
+	HelperTimeoutDuration = time.Duration(HelperTimeout) * time.Second
```

Comment:
HelperTimeout was initially a duration, but we moved away from that due to 460c376e45813f77285ea2cc6c23c7854a30807c - I'd prefer to keep one or the other, otherwise I feel like we will run into the same issues that that commit addressed.

Reply:
I disagree, this clearly states this is a `time.Duration`. Commit 460c376 fixed a different issue related with semantics where we had a Duration without the definition of the time unit set. One of the fixes could have been the one introduced in 460c376, the other could have been the change the of `TimeoutConfig`. As we are moving towards `context.Context` the `HelperTimeout` will likely be deleted.